api/helpers/gamestate/get-log.js

```diff
@@ -81,7 +81,7 @@ module.exports = {
 
         case MoveType.FIZZLE:
           return `The ${getFullCardName(
-            game.oneOff,
+            resolved,
```

Comment:
By the time a one-off fizzles, it is in the scrap so we need to use the `resolved` variable which represents which one-off just fizzled/resolved instead of `game.oneOff`